frontend/src/metabase/home/components/EmbedHomepage/EmbedHomepage.tsx

```diff
@@ -51,22 +55,55 @@ export const EmbedHomepage = () => {
   }, [plan]);
 
   const onDismiss = (reason: EmbedHomepageDismissReason) => {
-    dispatch(updateSetting({ key: "embedding-homepage", value: reason }));
+    if (reason === "dismissed-run-into-issues") {
```

Comment:
Not totally relate to this PR, but it's too easy to close the menu accidentally when moving the cursor into the menu slowly.
![image](https://github.com/metabase/metabase/assets/1937582/58f4c0b6-2ebc-41c9-9698-dfa279317273)


Could you add some close delay to the menu? This should make the experience better.
```ts
        <Menu trigger="hover" closeDelay={20}>
```

Reply:
Increased it to 200 as [the default](https://github.com/mantinedev/mantine/blob/v6/src/mantine-core/src/Menu/Menu.tsx#L80) is 100